packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -98,7 +98,7 @@ const GlobalHeader = ({ className }) => {
       >
         <div
           css={css`
-            height: 36px;
+            height: var(--global-header-height);
```

Comment:
i imagine we can eventually use this variable to offset the scroll to anchor link?

Reply:
Yep!